src/components/LHNOptionsList/LHNOptionsList.tsx

```diff
@@ -113,19 +159,30 @@ function LHNOptionsList({
     );
 
     return (
-        <View style={style ?? styles.flex1}>
-            <FlashList
-                indicatorStyle="white"
-                keyboardShouldPersistTaps="always"
-                contentContainerStyle={StyleSheet.flatten(contentContainerStyles)}
-                data={data}
-                testID="lhn-options-list"
-                keyExtractor={keyExtractor}
-                renderItem={renderItem}
-                estimatedItemSize={optionMode === CONST.OPTION_MODE.COMPACT ? variables.optionRowHeightCompact : variables.optionRowHeight}
-                extraData={[currentReportID]}
-                showsVerticalScrollIndicator={false}
-            />
+        <View style={[style ?? styles.flex1, shouldShowEmptyUI ? styles.emptyLHNWrapper : undefined]}>
+            {shouldShowEmptyUI ? (
+                <BlockingView
+                    animation={LottieAnimations.Fireworks}
+                    animationStyles={styles.emptyLHNAnimation}
+                    animationWebStyle={styles.emptyLHNAnimationWeb(isExtraSmallScreenHeight)}
```

Comment:
@tienifr Why do we need to add this prop?

Reply:
Please help to detail why `px unit works differently between native and web platforms`